src/databricks/labs/blueprint/paths.py

```diff
@@ -61,8 +61,11 @@ def __init__(self, ws: WorkspaceClient, path: str):
         StringIO.__init__(self)
 
 
-class WorkspacePath(Path):  # pylint: disable=too-many-public-methods
-    """Experimental implementation of pathlib.Path for Databricks Workspace."""
+DBPath = TypeVar("DBPath", bound="_DatabricksPath")
```

Comment:
```suggestion
P = TypeVar("DBPath", bound="_DatabricksPath")
```
because `DB` is "database"

Reply:
There's a different `P` already further down but I agree and will come up with something better.

Reply:
there's `T` :P isn't Java convention to use single-capital-letter for typevars?

Reply:
It was `T` that I was already using. 😂

Reply:
Fixed.